skyportal/utils/cosmology.py

```diff
@@ -0,0 +1,64 @@
+from astropy import cosmology
+from astropy import units as u
+
+from baselayer.log import make_log
+
+log = make_log('cosmology')
```

Comment:
I think this can safely be removed? Or is there a way to set log_level = DEBUG?

Reply:
Not sure whether removing or setting to debug is preferable but taking a quick look through the baselayer code, it doesn't seem like the concept of log levels are implemented yet like with the Python logging module. Maybe it's just subtly hidden somewhere, but if not it would probably be generally good to have. 

@stefanv Does baselayer logging have log levels?

Reply:
I think it's good (for posterity) to have a record in the logs about what cosmology is being used. It shows up only a boot time, not throughout a running SP.